dependencies.yaml

```diff
@@ -1017,6 +1018,12 @@ dependencies:
         packages:
           - dask-cuda==25.6.*,>=0.0.0a0
           - *numpy
+  test_python_narwhals:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - nvtx>=0.2.1,<0.2.12
```

Comment:
This isn't needed here because we're already getting this pinning from pylibcudf.
```suggestion
```